packages/auth/README.md

```diff
@@ -0,0 +1,51 @@
+[//]: # `{ "title": "@mintbase-js/auth", "order": 10 }`
```

Comment:
Adding a single line comment that contains metadata for the migration script to generate some nicer menu titles